ob-pattern-2/values.yaml

```diff
@@ -0,0 +1,248 @@
+        # Please see the official documentation on WSO2 Stream Processing based Performance Analysis and Resource recommendations
+        # (https://docs.wso2.com/display/SP440/Performance+Analysis+Results) and tune the limits according to your needs
+        # where necessary.
+        limits:
+          # The maximum amount of memory that should be allocated for a Pod
+          memory: "4Gi"
+          # The maximum amount of CPU that should be allocated for a Pod
+          cpu: "2000m"
+      imagePullPolicy: Always
+
+    wso2obbiDashboard:
+      # Hostname of the Open Banking Business Intelligence worker profile 
+      hostname: "wso2-obbi-dashboard"
+      # If a custom image must be used, uncomment 'dockerRegistry' and provide its value.
+      dockerRegistry: "docker.wso2.com"
+      imageName: "wso2-obbi-dashboard"
+      imageTag: "1.4.0"
+      # Number of deployment replicas
+      replicas: 1
+      minReadySeconds: 30
+      strategy:
+        rollingUpdate:
+          # The maximum number of pods that can be scheduled above the desired number of pods.
+          maxSurge: 1
+          # The maximum number of pods that can be unavailable during the update.
+          maxUnavailable: 0
+      livenessProbe:
+        # Number of seconds after the container has started before liveness probes are initiated.
+        initialDelaySeconds: 100
+        # How often (in seconds) to perform the probe.
+        periodSeconds: 10
+      # Indicates whether the container is ready to service requests.
+      readinessProbe:
+        # Number of seconds after the container has started before readiness probes are initiated.
+        initialDelaySeconds: 100
+        # How often (in seconds) to perform the probe.
+        periodSeconds: 10
+      resources:
+        # These are the minimum resource recommendations for running WSO2 Stream Processing based server profiles
+        # as per official documentation (https://docs.wso2.com/display/SP440/Installation+Prerequisites).
+        requests:
+          # The minimum amount of memory that should be allocated for a Pod
+          memory: "2Gi"
+          # The minimum amount of CPU that should be allocated for a Pod
+          cpu: "2000m"
+        # Please see the official documentation on WSO2 Stream Processing based Performance Analysis and Resource recommendations
+        # (https://docs.wso2.com/display/SP440/Performance+Analysis+Results) and tune the limits according to your needs
+        # where necessary.
+        limits:
+          # The maximum amount of memory that should be allocated for a Pod
+          memory: "2Gi"
+          # The maximum amount of CPU that should be allocated for a Pod
+          cpu: "2000m"
+      # Refer to the Kubernetes documentation on updating images (https://kubernetes.io/docs/concepts/containers/images/#updating-images)
+      imagePullPolicy: Always
+
+kubernetes:
+  # Name of Kubernetes service account
+  svcaccount: "wso2svc-account"
```

Comment:
@kaumini shall we change this parameter name to `serviceAccount`?

Reply:
@kdanW applies to pattern 1 as well.

Reply:
Fixed with 84268724a46bc3dc3b34b28b33f212be7087fb00